client/components/App.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Route } from 'react-router-dom';
+
+import Navbar from './Navbar';
+import Login from './Login';
+import Signup from './Signup';
+import Dashboard from './Dashboard';
+import Auth from './Auth';
+import { createUser, authenticateUser, logout } from '../actions/actions';
+
```

Comment:
This is the `<App />` component, one of two components that are connected to redux. This component takes care of all of the login stuff, it dispatches Redux actions when needed

Reply:
it also renders all of the necessary routes on the homepage